examples/applications/llamaindex-cassandra-sink/python/llamaindex_cassandra.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import List, Optional, Dict, Any
+
+import openai
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster
+from langstream import Sink, CommitCallback, Record
+from llama_index import VectorStoreIndex, Document
+from llama_index.vector_stores import CassandraVectorStore
+
+
+class LlamaIndexCassandraSink(Sink):
+
+    def __init__(self):
+        self.commit_cb: Optional[CommitCallback] = None
+        self.config = None
+        self.session = None
+        self.index = None
+
+    def init(self, config: Dict[str, Any]):
+        self.config = config
+        openai.api_key = config['openai-key']
+
+    def start(self):
+        cluster = Cluster(
+            cloud={
+                "secure_connect_bundle": "/app-code-download/python/secure-connect-bundle.zip",
```

Comment:
please add a comment to say that this is just an example and you should not store the secure connect bundle zip

Reply:
please create a agentContext variable to get the path for the application code

Reply:
it would be great to do as in the other agents and read the secure bundle as base64 encoded

I am telling this because everybody that is trying out LS already has configured the secure bundle that way in the resources file and they already have the secret
like here:

https://github.com/LangStream/langstream/tree/19b8b0b165a0d94e16fb388fd9e971c1405ca6c4/examples/applications/astradb-sink

Reply:
OK. Then I guess we can write a temp file and zip it from the bytes.

Reply:
Actually we can create in-mem file-like objects in python that should be work with the cassandra driver. I'll have a look.

Reply:
done. PTAL again